app/Models/Product.php

```diff
@@ -38,4 +38,14 @@ public function category()
         return $this->belongsTo(Category::class);
     }
 
+    public function orders()
+    {
+        return $this->belongsToMany(Product::class, 'product_order')->withTimestamps();
+    }
+
+    public function product_orders(){
```

Comment:
this relation is not good. 
it's not ok to define relation for pivot many to many tables